### PR TITLE
Charm authoring updates

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by cmars** [comment]
LGTM

File: src/en/authors-charm-store.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ In order to interact with the charm store you will need the latest
 [Ubuntu SSO account](https://login.ubuntu.com/+login), and you must have
 logged in to [Launchpad](https://launchpad.net/+login) at least once.
 
-## Login to the store
+## Log in to the store
+
+It is required that you first log in to the
+[Charmstore](https://jujucharms.com) before you attempt to log in via `charm
+login`. After logging in via your browser, you can use the `charm`
```
Comment (hatched) [inline src/en/authors-charm-store.md]: comma splice (I think) ;)

File: src/en/authors-charm-store.md [hunk 2]
```diff
@@ -31,6 +36,11 @@ following information:
  - `Two-factor auth` - If enabled, enter two-factor authentication (2FA)
 information. Otherwise <kbd>Return</kbd> for None
 
+Also note, if your group memberships change on Launchpad, you'll need to
```
Comment (hatched) [inline src/en/authors-charm-store.md]: ...Launchpad you'll...

**Comment by hatched** [comment]
:+1: Thanks for these changes, my comments are personal preference :)

File: src/en/authors-charm-store.md (hunk 2 above)
Comment (cmars) [inline src/en/authors-charm-store.md]: Seems like an Oxford comma ("if" is conjunctive enough, right?) but probably should `s/Also note,/Also note:/`.

File: src/en/authors-charm-store.md (hunk 1 above)
Comment (cmars) [inline src/en/authors-charm-store.md]: I think "After" could be considered conjunctive here, which would make this an Oxford comma.

**Comment by evilnick** [comment]
I approve these commas. :tangerine:

**Pull request merged**